packages/shared/src/components/notifications/NotificationItem.tsx

```diff
@@ -60,12 +62,16 @@ function NotificationItem({
       )}
     >
       {onClick && (
-        <button
-          type="button"
-          aria-label="Open notification"
-          className="absolute inset-0 z-0"
-          onClick={onClick}
-        />
+        <Link href={targetUrl} passHref>
+          {/* eslint-disable-next-line jsx-a11y/click-events-have-key-events, jsx-a11y/anchor-has-content, jsx-a11y/no-static-element-interactions */}
+          <a
+            aria-label="Open notification"
+            className="absolute inset-0 z-0"
+            onClick={onClick}
+            title="Open notification"
+            data-testid="openNotification"
```

Comment:
It seems you are not using this right? (the testId)

Reply:
I am using it in tests